samples/charts/area/line-boundaries.html

```diff
@@ -18,14 +18,14 @@
 		<div class="wrapper col-2"><canvas id="chart-3"></canvas></div>
 
 		<div class="toolbar">
-			<button onclick="toggleSmooth(this)">Smooth</button>
-			<button onclick="randomize(this)">Randomize</button>
+			<button id="toggleSmooth">Smooth</button>
```

Comment:
What the reason of this change?

Reply:
Linting JS in HTML attributes is not supported yet, so I had to move event handling in a `<script>` element.
I submitted a PR to add attributes linting support : https://github.com/BenoitZugmeyer/eslint-plugin-html/pull/87

Reply:
I really prefer the other syntax and was about to update other samples to remove these `addEventListener`.

Can we revert these changes?

Reply:
If the PR is accepted, yes. 
If not, we have 2 choices : 
* use `onclick()` and add a rule to ignore `no-undef-var` for /samples
* use `addEventListener`

Reply:
`no-undef-var: 0` in `samples/.eslintrc.yml` :)

Reply:
Alternatively (and probably better), we can disable this rule inline before each method:

```javascript
// eslint-disable-next-line no-undef-var
function randomize() {
//...
```